flow/src/test/java/com/sequenceiq/flow/event/FlowSelectors.java

```diff
@@ -0,0 +1,58 @@
+package com.sequenceiq.flow.event;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+import java.util.TreeMap;
+import java.util.stream.Collectors;
+
+import org.reflections.Reflections;
+import org.reflections.scanners.SubTypesScanner;
+
+import com.sequenceiq.flow.core.FlowEvent;
+
+public class FlowSelectors {
+
+    public void assertUniquenessInFlowEventNames(String packagePrefix) {
+        Reflections reflections = new Reflections(packagePrefix, new SubTypesScanner());
+        Set<Class<? extends FlowEvent>> eventClasses = reflections.getSubTypesOf(FlowEvent.class);
+        assertThat(eventClasses)
+                .withFailMessage("No FlowEvent subtypes found in package %s. Please check your 'packagePrefix' parameter.", packagePrefix)
+                .isNotEmpty();
+        Map<String, List<String>> names = new HashMap<>();
+        eventClasses.forEach(enumClass -> Arrays.stream(enumClass.getEnumConstants())
+                .forEach(item -> names.compute(item.event(), (enumName, pair) -> pair == null
```

Comment:
`pair` should be renamed; it still reflects the old implementation that was based on `Pair`.

Reply:
renamed a few vars